include/libdnf5/rpm/package.hpp

```diff
@@ -146,13 +146,15 @@ class LIBDNF_API Package {
     std::string get_arch() const;
 
     /// @return RPM package EVR (Epoch:Version-Release). If the Epoch is 0, it is omitted from the output.
+    /// @deprecated Use separate components or full NEVRA for consistent epoch display
```

Comment:
Do we really want to deprecate these?
I would be fine with leaving them on the API.

Reply:
I don't have strong opinion on this, my thought to deprecated is warning API users is better to use full nevra, but I'm good to leave them like it was if it is the case

Reply:
Reverted these changes as suggested.